Ds3/Helpers/Ds3ClientHelpers.cs

```diff
@@ -35,47 +35,30 @@ public class Ds3ClientHelpers : IDs3ClientHelpers
         private readonly int _objectTransferAttempts; //Negative number represents infinite number of retries
         private readonly int _jobRetries;
         private readonly int _jobWaitTime; //in minutes
-        private readonly long? _maximumFileSizeForAggregating;
 
         public Ds3ClientHelpers(IDs3Client client, int retryAfter = -1, int objectTransferAttempts = 5, int jobRetries = -1,
-            int jobWaitTime = 5, long? maximumFileSizeForAggregating = null)
+            int jobWaitTime = 5)
         {
-            this._client = client;
-            this._retryAfter = retryAfter;
-            this._objectTransferAttempts = objectTransferAttempts;
-            this._jobRetries = jobRetries;
-            this._jobWaitTime = jobWaitTime;
-            this._maximumFileSizeForAggregating = maximumFileSizeForAggregating;
+            _client = client;
+            _retryAfter = retryAfter;
+            _objectTransferAttempts = objectTransferAttempts;
+            _jobRetries = jobRetries;
+            _jobWaitTime = jobWaitTime;
         }
 
-        public IJob StartWriteJob(string bucket, IEnumerable<Ds3Object> objectsToWrite, long? maxBlobSize = null,
-            IHelperStrategy<string> helperStrategy = null)
+        public IJob StartWriteJob(string bucket, IEnumerable<Ds3Object> objectsToWrite, IHelperStrategy<string> helperStrategy = null, Ds3WriteJobOptions ds3WriteJobOptions = null)
```

Comment:
Rather than setting `ds3WriteJobOptions` to null by default, create a default options object.  This is what the java helper functions do, and it helps clean up the code later so that you don't have to check for null.  You just get the default value.